pkg/cover/templates/heatmap.html

```diff
@@ -138,12 +138,47 @@
       </ul>
     </div>
     <div style="display:inline-block; vertical-align: top">
-      <pre id="file-details-curr"></pre>
+      <div style="display:inline-block">
+        <form>
```

Comment:
Nit: at least for me, the form is display on the right to the table, which is rather unusual. On syzbot, we have them mostly on the left (e.g. https://20250108t131606-dot-syzkaller.uc.r.appspot.com/upstream/graph/fuzzing, https://20250108t131606-dot-syzkaller.uc.r.appspot.com/upstream/graph/crashes). On the top of the table would also seem somewhat more natural.

Reply:
I'll move it to the top. It was my attempt to save some space. But it is cheap... it was better to keep it top-left.